app/controllers/answers_controller.rb

```diff
@@ -37,6 +37,11 @@ def destroy
     end
   end
 
+  def accept
```

Comment:
We might want to also check that only the user that asked the question set an answer as accepted. Basically commend it to the cares of `Ownership concerns`